FlagsmithClient/src/main/java/com/flagsmith/internal/FlagsmithRetrofitService.kt

```diff
@@ -38,7 +55,8 @@ interface FlagsmithRetrofitService {
             requestTimeoutSeconds: Long,
             readTimeoutSeconds: Long,
             writeTimeoutSeconds: Long,
-        ): FlagsmithRetrofitService {
+            timeTracker: FlagsmithEventTimeTracker,
+        ): Pair<FlagsmithRetrofitService, Cache?> {
```

Comment:
In general, I prefer to avoid returning multiple objects from a single method. Do these need to be part of the same thing? Couldn't we pass the cache into the `create` method for example?

Reply:
I would prefer to pass the cache in myself, but to clear it we need to see the constructed Cache in the class that uses and owns it.

Don't forget this is an interface rather than a class so doesn't really own anything. I can't have a 'clearCache' method on it for example, which would have been nicer.

I can pass the Cache in if you like, to avoid returning the tuple, but I'm then bringing some of the Retrofit internals and logic into the Flagsmith class, which I was trying to avoid.

Reply:
Hmm... yeah, ok, seems like we've thought about it enough to justify it.